2-tycho-build.sh

```diff
@@ -4,11 +4,12 @@ if [ -z "$JENKINS_URL" ]; then
 fi
 
 mvn \
-  -f releng \
+  -f org.eclipse.xtext.p2.releng \
+  clean verify \
```

Comment:
I'd expect the goals to be at the end of the argument list. Why did you move them up?

Reply:
I usually prefer to have the goals initially, but I can move them down. Do you prefer that?

Reply:
Both variants are fine for me. It was more curiosity than a real thing.